include/sdf/World.hh

```diff
@@ -516,6 +516,14 @@ namespace sdf
     /// \param[in] _plugin Plugin to add.
     public: void AddPlugin(const Plugin &_plugin);
 
+    /// \brief Calculate and set the inertials for all the models in the world
+    /// object
+    /// \param[in] _config Custom parser configuration
+    /// \param[out] _errrors A vector of Errors objects. Each errors contains an
+    /// Error code and a message. An empty errors vector indicates no errors
```

Comment:
```suggestion
    /// \param[out] _errrors A vector of Errors objects. Each errors contains an
    /// Error code and a message. An empty errors vector indicates no errors
    /// \param[in] _config Custom parser configuration
```

Reply:
Corrected in [1f16e1f](https://github.com/gazebosim/sdformat/pull/1304/commits/1f16e1f0a5454acc5a30a4b86b8f273b1535eb95)